api/top-langs.js

```diff
@@ -44,6 +45,7 @@ module.exports = async (req, res) => {
       hide_title: parseBoolean(hide_title),
       card_width: parseInt(card_width, 10),
       hide_langs_below: parseFloat(hide_langs_below, 10),
+      hide_langs,
```

Comment:
Check out how i'm handling arrays 
https://github.com/anuraghazra/github-readme-stats/blob/03f55e809e417abcc0e4b462d52f3c3c52dd1a27/api/index.js#L47

Reply:
I just checked this online. It looks like the standard way of passing an array through url is `?hide=java&hide=html...` This seems to be supported by the current http framework being used also. So, now, I am passing the values like this:
```js
hide: (typeof hide == 'string') ? [hide] : hide,
// ?hide=java -> returns ['java']
// ?hide=java&hide=html -> returns ['java', 'html']
```

Reply:
Do you agree with this? 
Or should we just go with `?hide=['html','java']`

Reply:
Also, if we do `JSON.parse()` on a string that is not in correct json format, the card will not render

Reply:
Yes i know the standard way is like that, but i did not used it because i know in future we might wanted hide lots of options and the query string will get very messy and big, that's why i choose the direct JSON string approach also because it ensures the data is in JSON.

Reply:
Done. Updated the PR